tests/gem5/memory/test.py

```diff
@@ -41,10 +41,10 @@
 )
 
 simple_mem_params = [
-    ("inf-bandwidth", {"bandwidth": "0GB/s"}),
+    ("inf-bandwidth", {"bandwidth": "0GiB/s"}),
```

Comment:
I'm not certain these bandwidths should be changed. I know the reported bandwidth is in MB/s (power of 10).

The question is: Does the change you are making here have a functional difference?

Reply:
I looked into this and I think it should be fine to change. `("inf-bandwidth", {"bandwidth": "0GiB/s"}),` and `("low-bandwidth", {"bandwidth": "1MiB/s"}),` are passed into `tests/gem5/memory/simple-run.py` as bandwidth arguments, which are then used in `class MyMem(SimpleMemory)`. In SimpleMemory.py, bandwidth uses `Param.MemoryBandwidth()`, which calls `convert.toMemoryBandwidth()`. 

`convert.toMemoryBandwidth` uses `toBinaryFloat()`, which uses `binary_prefixes` in `convert.py`, which means that it converts base 10 prefixes to base 2 numbers.